protobuf/grpc_service.proto

```diff
@@ -894,7 +894,8 @@ message InferStatistics
   //@@  .. cpp:var:: StatisticDuration success
   //@@
   //@@     Cumulative count and duration for successful inference
-  //@@     request.
+  //@@     request. The "success" count and cumulative duration includes
+  //@@     cache hits.
```

Comment:
What are cache hits? May be write a 1-2 lines about it and link to the doc.

Reply:
Maybe instead of mentioning the same in every field, just clarify what cache hit means and specify what fields also accumulate cache stats